pkgs/by-name/vo/vors/package.nix

```diff
@@ -36,21 +37,29 @@ stdenv.mkDerivation (finalAttrs: {
     makeWrapper
```

Comment:
I think `makeWrapper` is not used at all, even in the 3.1.0 Nix expression. Could you please remove it in a separate commit before the version bump update? Also, this is the place to add `installShellFiles`.